tests/ci/nightly_gpu.yml

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
do you think you will have to change the name to not get a conflict?

Reply:
the pipeline name is the conflict not the yml.
When the PR is included in staging, then the pipeline name is changed to bp-nightly_gpu and the yml file remains the same.